front-end/bundle-configuration/default/runtime-config.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+export APPLICATION_SECRET=PPjOW0n2aV?s@6RdiNV@7/5xJhiiKTzk[VdHjkU9YHit8sLHoJ1rp0DCCn6b=lXt
```

Comment:
Yes, we certainly can add a comment. Note that we need to update the project anyway once the corresponding ConductR video is hosted on lightbend.com.